src/main/java/com/hazelcast/gcp/GcpAuthenticator.java

```diff
@@ -59,6 +64,10 @@ String refreshAccessToken(String privateKeyPath, long currentTimeMs) {
             String body = createBody(privateKeyPath, currentTimeMs);
             String response = callService(body);
             return parseResponse(response);
+        } catch (FileNotFoundException e) {
+            LOGGER.severe(String.format("Private key json file not found. Please ensure you have stored the json file"
```

Comment:
Do we need to both log error and throw an exception? Maybe just put this comment into the exception message? I guess it will be printed to the user in the exception stacktace?